ExtractorUtils/ExtractorRunner.cs

```diff
@@ -245,8 +247,7 @@ void CancelKeyPressHandler(object sender, ConsoleCancelEventArgs eArgs)
                 ConfigurationException? exception = null;
                 try
                 {
-                    bool usesRemoteConfig = false;
-                    if (options.AllowRemoteConfig && options.Config == null)
+                    if (options.AllowRemoteConfig && options.Config == null || usesRemoteConfig)
```

Comment:
I think the `usesRemoteConfig` here will always evaluate to `false` so checking it in the conditional might be redundant.

Reply:
It won't any more, I moved the bool outside the loop.

First iteration of the loop it is set to `true`.

Reply:
I didn't notice the loop, my bad.